### PR TITLE
[doc][ybm] Changes for pricing plans #20900

### PR DESCRIPTION
**Pull request by ddhodge**
Changes for pricing plans
CLOUDGA-21241
CLOUDGA-21234

@netlify /preview/yugabyte-cloud/

### CODE REVIEW

File: docs/content/v2.20/explore/fault-tolerance/handling-region-failures.md [hunk 1]
```diff
@@ -69,11 +69,11 @@ To simulate the failure of the 3rd region locally, you can just stop the third n
 {{</nav/panel>}}
 
 {{<nav/panel name="anywhere">}}
-{{<note>}} To stop a node in YB Anywhere, see [YBA - Manage nodes](../../../yugabyte-platform/manage-deployments/remove-nodes/#start-and-stop-node-processes). {{</note>}}
+{{<note>}} To stop a node in YugabyteDB Anywhere, see [Manage nodes](../../../yugabyte-platform/manage-deployments/remove-nodes/#start-and-stop-node-processes). {{</note>}}
 {{</nav/panel>}}
 
 {{<nav/panel name="cloud">}}
-{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in YB Managed. {{</note>}}
+{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in YugabyteDB Managed. {{</note>}}
```
Comment (aishwarya24) [inline docs/content/v2.20/explore/fault-tolerance/handling-region-failures.md:76]:
```suggestion
{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in YugabyteDB Aeon. {{</note>}}
```

File: docs/content/v2.20/explore/fault-tolerance/handling-region-failures.md [hunk 2]
```diff
@@ -103,11 +103,11 @@ To simulate the failure of the secondary region locally, you can just stop the s
...
```
Comment (aishwarya24) [inline docs/content/v2.20/explore/fault-tolerance/handling-region-failures.md:110]:
```suggestion
{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in YugabyteDB Managed. {{</note>}}
```
```suggestion
{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in YugabyteDB Aeon. {{</note>}}
```

File: docs/content/v2.20/explore/fault-tolerance/handling-region-failures.md [hunk 3]
```diff
@@ -137,11 +137,11 @@ To simulate the failure of the primary region locally, you can just stop the fir
...
```
Comment (aishwarya24) [inline docs/content/v2.20/explore/fault-tolerance/handling-region-failures.md:144]:
```suggestion
{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in YugabyteDB Aeon. {{</note>}}
```

File: docs/content/v2.20/explore/fault-tolerance/handling-node-upgrades.md [hunk 4]
```diff
@@ -69,11 +69,11 @@ To take a node offline locally, you can just stop the node.
 {{</nav/panel>}}
 
 {{<nav/panel name="anywhere">}}
-{{<note>}} To stop a node in YB Anywhere, see [YBA - Manage nodes](../../../yugabyte-platform/manage-deployments/remove-nodes/#start-and-stop-node-processes). {{</note>}}
+{{<note>}} To stop a node in YugabyteDB Anywhere, see [Manage nodes](../../../yugabyte-platform/manage-deployments/remove-nodes/#start-and-stop-node-processes). {{</note>}}
 {{</nav/panel>}}
 
 {{<nav/panel name="cloud">}}
-{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in [YB Managed](/preview/yugabyte-cloud/) for an upgrade. {{</note>}}
+{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in [YugabyteDB Managed](/preview/yugabyte-cloud/) for an upgrade. {{</note>}}
```
Comment (aishwarya24) [inline docs/content/v2.20/explore/fault-tolerance/handling-node-upgrades.md:76]:
```suggestion
{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to stop a node in [YugabyteDB Aeon](/preview/yugabyte-cloud/) for an upgrade. {{</note>}}
```

File: docs/content/v2.20/explore/fault-tolerance/handling-node-upgrades.md [hunk 5]
```diff
@@ -119,11 +119,11 @@ To simulate bringing back a node online locally, you can just start the stopped
 {{</nav/panel>}}
 
 {{<nav/panel name="anywhere">}}
-{{<note>}} To restart a node in YB Anywhere, see [YBA - Manage nodes](../../../yugabyte-platform/manage-deployments/remove-nodes/#start-and-stop-node-processes). {{</note>}}
+{{<note>}} To restart a node in YugabyteDB Anywhere, see [Manage nodes](../../../yugabyte-platform/manage-deployments/remove-nodes/#start-and-stop-node-processes). {{</note>}}
 {{</nav/panel>}}
 
 {{<nav/panel name="cloud">}}
-{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to restart a node in [YB Managed](/preview/yugabyte-cloud/).  {{</note>}}
+{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to restart a node in [YugabyteDB Managed](/preview/yugabyte-cloud/).  {{</note>}}
```
Comment (aishwarya24) [inline docs/content/v2.20/explore/fault-tolerance/handling-node-upgrades.md:126]:
```suggestion
{{<note>}} Reach out [YugabyteDB support](https://support.yugabyte.com) to restart a node in [YugabyteDB Aeon](/preview/yugabyte-cloud/).  {{</note>}}
```

**Review by aishwarya24 — COMMENTED** [review]
some comments

File: docs/content/preview/admin/ysqlsh-meta-commands.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ Similarly to psql, ysqlsh provides a number of meta-commands that make ysqlsh mo
 
 {{< note title="Cloud shell" >}}
 
-For security reasons, the YugabyteDB Managed cloud shell only has access to a subset of meta-commands. With the exception of read-only access to the `/share` directory to load the sample datasets, commands that access the filesystem don't work in cloud shell.
```
Comment (rangasayeelc-yb) [inline docs/content/preview/admin/ysqlsh-meta-commands.md:19]: 'Similarly' -> 'Similar to psql'

**Review by rangasayeelc-yb — APPROVED** [review]
eyeballed the changes across 20-25 files. I can't seem to figure out how to effectively use additional tooling to preview each page.

**Review by rangasayeelc-yb — COMMENTED** [review]
@ddhodge these are the naming changes. Where can I find the pricing plan updates?

File: docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-costs.md [hunk 7]
```diff
@@ -20,36 +32,23 @@ Your bill is calculated based on your usage of the following dimensions:
...
-| Backup storage | 100 GB |
-| Data transfer – Same Region | 1000 GB |
-| Data transfer – Cross Region (APAC) | 10 GB |
-| Data transfer – Cross Region (Other regions) | 10 GB |
-| Data transfer – Internet | 10 GB |
-
-You can see the approximate cost for your vCPUs when [creating](../../cloud-basics/create-clusters/) and [scaling](../../cloud-clusters/configure-clusters/) clusters, as shown in the following illustration. **+ Usage** in this context refers to any potential overages from exceeding the free allowances for disk storage, backup storage, and data transfer.
-
-![Cluster Settings Edit Infrastructure](/images/yb-cloud/cloud-clusters-settings-edit.png)
+The following calculations reflect the Standard plan rate card.
```
Comment (rangasayeelc-yb) [inline docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-costs.md:35]: @ddhodge can you please link to the pricing page from here ('Standard plan rate card')? Or add a link for more information, for example, 'For more information on plan based pricing, see <yugabyte.com/pricing>'

File: docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-costs.md [hunk 8]
```diff
@@ -1,17 +1,29 @@
 ---
 title: Cluster costs
 linkTitle: Cluster costs
-description: YugabyteDB Managed cluster configuration costs.
-headcontent: YugabyteDB Managed cluster configuration costs
+description: YugabyteDB Aeon cluster configuration costs.
+headcontent: Plan-based cluster pricing
```
Comment (rangasayeelc-yb) [inline docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-costs.md:5]: @ddhodge pricing is for the account, not per cluster or cluster based. Can we remove this sub-heading?

File: docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-costs.md [hunk 9]
```diff
@@ -1,17 +1,29 @@
...
     parent: cloud-admin
     weight: 400
 type: docs
 ---
 
-There are no set-up charges or commitments to begin using YugabyteDB Managed. At the end of the month, you are automatically charged for that month's usage. You can view your charges for the current billing period at any time by selecting the **Invoices** tab on the **Usage & Billing** page. Refer to [Manage your billing profile and payment method](../cloud-billing-profile/).
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../cloud-billing-costs/" class="nav-link active">
+      Plan Pricing
```
Comment (rangasayeelc-yb) [inline docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-costs.md:17]: rename to 'Plan based Pricing'

**Review by rangasayeelc-yb — CHANGES_REQUESTED** [review]
@ddhodge found the pricing changes, made a few suggestions on the same.
Renaming changes look good - eyeballed about 25 files but did not check everything

**Pull request merged**